src/templates/default/_includes/dashboard_tablist.html

```diff
@@ -22,4 +22,10 @@
 		{% endif %}
 	</li>
 	{% endif %}
+
+	{% if user.is_superuser %}
+	<li class="{% if active == 'admin' %}active{% endif %}">
+		<a href="{% url 'review_change' %}">{% trans 'Change Review' %}</a>
```

Comment:
Can we call it `Review Stages`? "Change Review" seems a little bit imprecise here.

Reply:
Thanks for your advise. To rename, I push a new commit db4fea727ef44bd3893ad83f52f070e19cdd20b0. **Review Stages** is indeed more precise.